src/index.ts

```diff
@@ -33,6 +32,8 @@ export default class IntegrationTestEnvironment extends NodeEnvironment {
         await this.global.ceramicClient.close();
         this.global.ceramicClient = null
 
+        // @ts-ignore
+        await this.global.ipfs.stop();
```

Comment:
I actually tried this last week, and it didn't work.  What I believe is happening is that `ipfs.stop()` on the http client seems to actually send a shutdown command to the remote ipfs node and make it shut down, it doesn't just tear down the client.  Since this teardown() function runs between every test, that means after the first test the ipfs node gets shut down and never starts back up.

Did you try testing this locally in the `local_client-public` or `local_node-private` configurations?

Reply:
Yes, I noticed it a minute after PR. Sometimes js-ipfs design decisions surprise me.